src/main/java/org/broadinstitute/consent/http/resources/DaaResource.java

```diff
@@ -222,4 +222,60 @@ public Response sendDaaRequestMessage(
       return createExceptionResponse(e);
     }
   }
+
+  @POST
+  @Consumes(MediaType.APPLICATION_JSON)
+  @RolesAllowed({ADMIN, CHAIRPERSON})
+  @Path("/bulk/{daaId}")
+  public Response bulkAddUsersToDaa(
+      @Auth AuthUser authUser,
+      @PathParam("daaId") Integer daaId,
+      String json) {
+    try {
+      User authedUser = userService.findUserByEmail(authUser.getEmail());
+      List<User> users = userService.findUsersInJsonArray(json, "users");
+      if (authedUser.hasUserRole(UserRoles.SIGNINGOFFICIAL)) {
+        for (User user : users) {
+          if (!Objects.equals(authedUser.getInstitutionId(), user.getInstitutionId())) {
+            return Response.status(Status.FORBIDDEN).build();
+          }
+        }
```

Comment:
The ticket isn't clear on this but I think the original intention for this was to query for all users in the SO's institution instead of accepting input of user ids. It would simplify the logic since we wouldn't need that extra parameter or check for an invalid request. This approach, however, does give us some additional flexibility in that it gives the UI the ability to select individual users and apply the DAA to only those selected.